Sources/NIOTestUtils/NIOThreadPoolTaskExecutor.swift

```diff
@@ -0,0 +1,92 @@
+#if compiler(>=6)
+
+import NIOPosix
+
+/// Run a `NIOThreadPool` based `TaskExecutor` while executing the given `body`.
```

Comment:
Let's clarify in this documentation that this is a `TaskExecutor`, but not a _serial_ executor.

Reply:
Done